source/geod24/LocalRest.d

```diff
@@ -421,12 +434,11 @@ public final class RemoteAPI (API) : API
                             filter = filter_api;
                         },
                         (Response res) {
-                            scheduler.pending = res;
-                            scheduler.waiting[res.id].c.notify();
+                            handle(res);
```

Comment:
I think you could just pass `handle!Response, handle!Command` then

Reply:
what

Reply:
Ah you mean `&handle!Response`

Reply:
Yup, sorry

Reply:
I wanted to do it just now, but then I realized in the second commit we add a bit of logic here.